### PR TITLE
Update balance(header)

### PR DESCRIPTION
**Pull request by valerijaslu**

### CODE REVIEW

File: src/common/utils.js [hunk 1]
```diff
@@ -12,6 +12,20 @@ export function createElementsFromString(str) {
   return result;
 }
 
+export function replaceFirstChild(target, newChild) {
+  if (target.firstChild) {
+    target.removeChild(target.firstChild);
+
+    if (target.firstChild) {
```
Comment (Stkulibaba) [inline src/common/utils.js]: Let`s just append new child if there is false in the first IF statement

File: src/components/header/header.js [hunk 2]
```diff
@@ -23,15 +24,40 @@ export default class Header {
       }
       header = createElementsFromString(usersHeader(headersProps));
     }
-    const screenWithHeader = target.appendChild(header);
+    if (target.firstChild) {
```
Comment (Stkulibaba) [inline src/components/header/header.js]: As i mentioned before we can use 'replaceFirstChild' without check on target.firstChild

File: src/components/header/header.js [hunk 3]
```diff
@@ -23,15 +24,40 @@ export default class Header {
...
+      header.querySelector('.header-content__switch-mode-button').addEventListener('click', () => {
         switchMode(props);
       });
     }
-    target.querySelector('.exit-ico').addEventListener('click', () => { logout(props.router); });
+    header.querySelector('.exit-ico').addEventListener('click', () => {
+      logout(props.router);
+    });
+
+    // to remove
```
Comment (Stkulibaba) [inline src/components/header/header.js]: Here is comment "to remove"
When should it be done?

File: src/components/header/header.js [hunk 4]
```diff
@@ -23,15 +24,40 @@ export default class Header {
...
+    }
+    //
+
+    this.unsubscribe = eventBus.subscribe('onBalanceChange', this.onBalanceChange.bind(this, target, props));
+
+    return header;
+  }
+
+  onBalanceChange(target, props, balance) {
+    setCookie('balance', balance, 365);
```
Comment (Stkulibaba) [inline src/components/header/header.js]: I guess it is not the right place to update cookie. So, in general Header depends on cookie and the right event change should looks like 
BalanceUpdate -publish balance update-> CookieUpdate -publish cookie update-> Component react on cookie update and do not react on BalanceUpdate

File: src/components/header/header.js (hunk 3 above)
Comment (valerijaslu) [reply src/components/header/header.js]: I think, when we will have real opportunity to test balance update

**Pull request merged**